test/src/apiSurfaceTests.cpp

```diff
@@ -118,11 +120,11 @@ TEST_F(APISurfaceTests, rgl_entity_create_destroy)
 	ASSERT_RGL_SUCCESS(rgl_entity_destroy(entity));
 
 	// Double destroy
-	EXPECT_RGL_INVALID_OBJECT(rgl_entity_destroy(entity), "Object does not exist: Entity");
+	EXPECT_RGL_OBJECT_NOT_EXISTS(rgl_entity_destroy(entity), "Object does not exist: Entity");
 
 	// Invalid destroy
 	EXPECT_RGL_INVALID_ARGUMENT(rgl_entity_destroy(nullptr), "entity != nullptr");
-	EXPECT_RGL_INVALID_OBJECT(rgl_entity_destroy((rgl_entity_t) 0x1234), "Object does not exist: Entity 0x1234");
+	EXPECT_RGL_OBJECT_NOT_EXISTS(rgl_entity_destroy((rgl_entity_t) 0x1234), "Object does not exist: Entity 0x1234");
 }
 
 TEST_F(APISurfaceTests, rgl_entity_set_pose)
```

Comment:
Not to Ur PR, but:
`rgl_entity_set_pose` Is terribly wrong named.
Pose in 3d envs, and CG means something very specific, and as far as I know, we are using skinned meshes, which can contain poses.

Should be renamed to FE:
`rgl_entity_set_local_transform`